src/deadline_worker_agent/capabilities.py

```diff
@@ -136,10 +164,7 @@ def _get_gpu_memory(*, verbose: bool = True) -> int:
         return 0
     output = output_bytes.decode().strip()
 
-    mem_per_gpu: list[int] = []
-    for line in output.splitlines():
-        mem_mib = int(line.replace("MiB", ""))
-        mem_per_gpu.append(mem_mib)
+    mem_per_gpu = _parse_gpu_memory(output)
```

Comment:
```suggestion
    mem_per_gpu = _parse_gpu_memory(output, verbose)
```

I think we will want to pass on the `verbose` option when we call this function

Reply:
Added `verbose` parameter in e78b9da.